bindings/python/iota_sdk/types/block/wrapper.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import Dict
+from dacite import from_dict
+from iota_sdk.types.block.block import Block
+from iota_sdk.types.common import HexStr, json
+from iota_sdk.types.signature import Signature
+from iota_sdk.utils import Utils
+
+
+@json
+@dataclass
+class BlockWrapper:
+    """A block wrapper type that can hold either a `BasicBlock` or a `ValidationBlock`.
+    Shared data is stored alongside such a block in the `BlockHeader` and `Signature` fields.
+
+    Attributes:
+        header: The Block header.
+        block: Holds either a `BasicBlock` or a `ValidationBlock`.
+        signature: The Block signature.
+    """
+    header: Header
```

Comment:
This would require a `header` field to be present which is not the case, needs to be flattened somehow

Reply:
I removed the header ... do you think flattening is worth the effort?

Reply:
Probably not